src/radioButtonGroup/index.js

```diff
@@ -33,7 +35,7 @@ class RadioButtonGroup extends React.Component {
         <div className={classNames}>
           {/* eslint-disable jsx-a11y/click-events-have-key-events */ }
           { /* eslint-disable jsx-a11y/no-static-element-interactions */ }
-          <div className={cx(styles['each-check'], { inline })} onClick={() => { this.handleCheckListChange(option.label); }}>
+          <div className={cx(styles['each-check'], { inline })} onClick={(e) => { this.handleCheckListChange(e, option); }}>
             <label className={cx(styles['customized-radio'], theme['customized-radio'])}>
               <label className={cx('inner', { checked: (option.label === this.state.currentlyActive) ? 'active' : '' })}>
```

Comment:
extract the checked option before return statement. Basically make this more readable.

Reply:
Extracted the checked option before return statement.